### ISSUE TITLE
Can we use the "List" view to highlight blocks with issues?

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Checklists#783]
People sometimes tell us its hard to see which block has issues such as missing ALT text.

Could we highlight those blocks in the "List" view.
![Screenshot 2024-10-01 at 10 56 27 AM](https://github.com/user-attachments/assets/0e0c791f-b952-43e5-985c-37ea3ceb0a79)

**Comment by rizaardiyanto1412** [comment]
![image](https://github.com/user-attachments/assets/cb3ef288-edc4-402e-bf23-54fb75de78ac)
Applied in 3 checklists now 
- All images have Alt text*
- characters in Alt text*
- All links use a valid format

Not all checklist make sense to have this

**Comment by stevejburge** [comment]
Woah, @rizaardiyanto1412. That's impressive work!
